athena_cfn/athena-bucket-cfn.yml

```
@@ -0,0 +1,247 @@
+# Purpose:  Expose Metadata from a single Quilt bucket in Athena
+#===============================================================================
+# https://aws.amazon.com/cloudformation/resources/templates/
+# https://catalog.us-east-1.prod.workshops.aws/workshops/9981f1a1-abdc-49b5-8387-cb01d238bb78/en-US/20-howtostart/201-self-paced/2013-cloudformation
+# https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/walkthrough-crossstackref.html
+
+AWSTemplateFormatVersion: "2010-09-09"
+Description: |
+  Configure Athena tables for a Quilt Bucket
+
+Parameters:
+  QuiltAthenaStack:
+    Description: Parent stack defining Athena Database, Workgroup and Bucket
+    Type: String
+    MinLength: 1
+    AllowedPattern: '[a-zA-Z][-a-zA-Z0-9]*'
+  QuiltBucket:
+    Description: Name of an S3 Bucket containing Quilt packages
+    Type: String
+    MinLength: 1
```

Might make sense to set `MaxLength` as well:
```suggestion
    MinLength: 1
    MaxLength: 128
    AllowedPattern: '[a-zA-Z][-a-zA-Z0-9]*'
  QuiltBucket:
    Description: Name of an S3 Bucket containing Quilt packages
    Type: String
    MinLength: 3
    MaxLength: 63
```